### PR TITLE
Remove practices on sensitive information

### PR DESCRIPTION
**Pull request by adborden**
https://github.com/18F/aws-admin/issues/92
https://github.com/18F/handbook/issues/2360

This content has been merged into the [TTS Handbook][1].

[1]: https://github.com/18F/handbook/pull/2518

### CODE REVIEW

File: practice.md [hunk 1]
```diff
@@ -110,27 +110,6 @@ When an 18F project has a non-standard license status (e.g. it's a fork of a pre
 
 All 18F team members are expected to make new source code repositories public from the time of creation. This means we often publish drafts in our repos that may change substantially. If you're interested in learning more about the contents of a repo, email 18F@gsa.gov and we'll direct you to the right person or team.
 
-## Protecting sensitive information
```
Comment (afeld) [inline practice.md:113]: Can we link to the Handbook?

**Review by afeld — CHANGES_REQUESTED** [review]

File: practice.md (hunk 1 above)
Comment (adborden) [reply practice.md:113]: We can, but I don't even think it's relevant for the Open Source Policy. Here's what I'm thinking _might_ make sense:

> ## Protecting sensitive information
>
> There are several types of information that should not be included in open source repositories. See the [TTS Handbook for how to handle sensitive information](https://handbook.tts.gsa.gov/sensitive-information/).

But IMHO it seems unnecessary to have an entire second level header for a single link. And wouldn't this section/sentence disappear when the rest of the practices.md document are migrated to the Handbook?

I'm also looking at other content in this doc and I don't see a great place to link it. Do you have a suggestion? I'd rather leave it out altogether.

Comment (afeld) [reply practice.md:113]: Yep, that suggestion works for me. I've linked to that heading from a number of places over the years, so I'd prefer to keep it.

Comment (adborden) [reply practice.md:113]: Okay, I left the section as a breadcrumb.

**Review by afeld — APPROVED** [review]

**Pull request merged**